core/src/main/java/org/elasticsearch/index/query/QueryBuilders.java

```diff
@@ -844,6 +850,65 @@ public static PercolatorQueryBuilder percolatorQuery(String documentType, String
                 indexedDocumentRouting, indexedDocumentPreference, indexedDocumentVersion);
     }
 
+    public static void registerBuiltinBuilders(SearchModule searchModule, NamedWriteableRegistry namedWriteableRegistry) {
```

Comment:
I had a look, I think I still like not having a Registrar for each type of thing that we have to register. I remember a while ago we had IndicesQueriesRegistry for instance and we got rid of it in favour of having all the registration happening in the SearchModule. I believe we don't want to go back that path.